frontend/packages/operator-lifecycle-manager/src/components/index.tsx

```diff
@@ -114,6 +114,11 @@ export const providedAPIForModel = (csv: ClusterServiceVersionKind, model: K8sKi
 
 export const parseALMExamples = (csv: ClusterServiceVersionKind) => {
   try {
+    if (new URLSearchParams(window.location.search).has('useInitializationResource')) {
+      return JSON.parse(
+        `[${csv?.metadata?.annotations?.['operatorframework.io/initialization-resource']}]` ?? '[]',
+      );
```

Comment:
Parse it as-is and then return it as an array.

```suggestion
      const resource = JSON.parse(csv?.metadata?.annotations?.['operatorframework.io/initialization-resource']);
      return [resource];
```